tests/int_test.ml

```diff
@@ -0,0 +1,28 @@
+open InfiniteJest
+open InfiniteJest.Test
+open Int
+
+let suite = 
+    describe "Int" (fun () -> [
+        test "pow" (fun () -> 
+            pow 2 1 |> Expect.toEqual 2
+        );
+        test "pow - raise" (fun () ->
+            (fun () -> pow 0 0) |> Expect.toRaise
+        );
+        test "pow - negative" (fun () -> 
+            (fun () -> pow 1 (~-1)) |> Expect.toRaise
+        );
+        test "fromString" (fun () ->
+            fromString "233" |> Expect.toEqual (Some 233)
+        );
+        test "fromString - None" (fun () ->
+            fromString "23a" |> Expect.toEqual (None)
+        );
+        test "toBinaryString - 1" (fun () ->
+            toBinaryString 1 |> Expect.toEqual ("0b1")
+        );
+        test "toBinaryString - 32" (fun () ->
+            toBinaryString 32 |> Expect.toEqual ("0b100000")
+        )
+    ])
```

Comment:
This whole file seems to be using 4-space indent. Or maybe ocp-indent is screwing up. Either way, we should stick with 2-space indents.